### PR TITLE
Add language option for Innoextract installation.

### PR DESCRIPTION
**Pull request by Kzimir**
## Description
Fix issue #427 

Create a function which search if the gog installer has the possibility to extract specific language.

## Checklist
 
 - [X] _CHANGELOG.md_ was updated (**format**: - Change made (thanks to Kzimir))

### CODE REVIEW

File: minigalaxy/installer.py [hunk 1]
```diff
@@ -334,3 +335,28 @@ def _mv(source_dir, target_dir):
...
+        if not line.startswith(' -'):
+            continue
+        languages.append(line[3:])
+    for lang in languages:
+        if "-" in lang:  # lang must be like "en-US" only.
+            if Config.get("lang") == lang[0:2]:
+                arg = "--language={}".format(lang)
+                break
+            else:
+                arg = "--language=en-US"
```
Comment (sharkwouter) [inline minigalaxy/installer.py:360]: I would leave arg empty if the language isn't found. Then the default should be used, right?

**Review by sharkwouter — CHANGES_REQUESTED** [review]
Thanks for this! This is something we really need. I do think it is a bit risky, though. It could break some things when a language doesn't quite match or partially matches. I would like to see some tests for this.

In this case you would have to get the output of the ``--list-languages`` option for one or more installers and then create some tests with mock the output of ``subprocess.Popen`` in this case. You'd test what the output of the lang_install function is with different configured languages.

File: minigalaxy/installer.py (hunk 1 above)
Comment (Kzimir) [reply minigalaxy/installer.py:360]: No. As i explained in #427, in arg is empty a random language is set.
For example with XCom Enemy Unknow, without "arg", the game has Asiatic language...
That's why i force to use en_US by default.

**Comment by sharkwouter** [comment]
Thanks for this PR. Sorry for taking forever. I've been in between jobs and moved. I should have a bit more time now to work on Minigalaxy, but will do a release first.

**Pull request merged**